### DISCUSSION TITLE
Response times

### DISCUSSION THREAD

**Discussion by dogotcat05** [sebr/bhyve-home-assistant#71] [General]
Hi there.

Awesome integration!

One thing though, are you able to fine tune the response times, as it can take up to 5 seconds or more for the sprinkler to turn on or off based off a entity change in home assistant.

I’m using the Orbit B-hyve Smart Hose Tap Timer with WiFi Hub
Model Number
96504

Works well, just could be quicker

**Comment by sebr** [comment]
Unfortunately not, this is a by product of the b-hyve hardware / software implementation. You will see that the same lag exists when using the official app.
